index.html

```diff
@@ -0,0 +1,396 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      youtube-page
+    </title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;1,100&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div class="header">
+      <div class="right-section">
+        <img class="hamburger-menu" src="header-icons/hamburger-menu.svg" alt="">
+       <div class="youtube-logo-container">
+        <img class="youtube-logo"src="header-icons/youtube-logo.svg" alt="">
+        <div class="country-symbol">
+          KE
+        </div>
+      </div>
+      </div>
+      <div class="middle-section"> 
+        <input class="search-bar" type="text" placeholder="Search">
+        <div class="search-container">
+          <img class="search-icon" src="header-icons/search-icon.svg" alt="">
+          <div class="tooltip2">search</div>
+        </div>
+        <div class="voice-search-container">
+          <img class="voice-search-icon" src="header-icons/voice-search-icon.svg" alt="">
+          <div class="tooltip2">
+            search with your voice
+          </div>
+        </div>
+      </div>
+      <div class="left-section">
+        <div class="upload-container">
+          <img class="upload-icon" src="header-icons/upload.svg" alt="">
+          <div class="tooltip2">upload</div>
+        </div>
+        <div class="notifications-container">
+          <img class=" notifications-icon" src="header-icons/notifications.svg" alt="">
+          <div class="tooltip2"> notifications</div>
+        </div>
+        <img class="youtube-apps" src="header-icons/youtube-apps.svg" alt=""> 
+      </div>
+    </div>
```

Comment:
Please use li tag and ul tag to list the items on this section

Reply:
- [ ] Kindly make your request specific and straight to the point. For instance you said 'Please use li tag and ul tag to list the items on this section'. I think you should also give the reason why to change it. Being more specific will help avoid review loops. :smile: